packages/signers/package.json

```diff
@@ -38,6 +38,7 @@
     "@noble/ed25519": "^2.1.0",
     "@noble/hashes": "^1.5.0",
     "@noble/secp256k1": "^2.3.0",
+    "@sovereign-sdk/universal-wallet-wasm": "workspace:^",
```

Comment:
This has a ripple affect and will make WASM compulsory when using `@sovereign-sdk/web3` again

Lets do what we're doing in the `serializers` package where we export 2 entry points, one that requires WASM and one that doesnt

So we'll need to:
- Add a `signers/src/wasm.ts` and export `Eip712` signer from there instead
- Update `signers/tsup.config.ts` to have 2 entrypoints (see `serializers/tsup.config.ts` if needed)
- Update `signers/package.json` and its `exports` field to export the wasm entrypoint, (see `serializers/package.json` if needed)